.pre-commit-config.yaml

```diff
@@ -81,7 +81,7 @@ repos:
         args:
           - --skip=logs/**,data/**
           # required for plotext.hist()
-          - --ignore-words-list=hist
+          - --ignore-words-list=hist,ba
```

Comment:
can you add a comment why this is added? and also modify the comment above to: `# hist: required for plotext.hist()`

Reply:
Added in [bc272b2](https://github.com/ArneBinder/pie-utils/pull/28/commits/bc272b294297e039a4ad29a2e4a68ea42a7af9f0)